examples/sveltekit/src/routes/api/graphql.ts

```diff
@@ -1,6 +1,6 @@
 import { useGraphQlJit } from '@envelop/graphql-jit';
 import { createServer } from '@graphql-yoga/common';
-import type { RequestEvent } from '@sveltejs/kit/types/hooks';
+import type { RequestEvent } from '@sveltejs/kit/types/private';
```

Comment:
I came across this PR after running into issues getting Yoga and SvelteKit working, and figured I'd leave a note on how I fixed the private-`type`-import issue for my own app; you can avoid using any private APIs by defining `RequestEvent` like this

```ts
import type { RequestHandler } from './graphql.d';

type RequestEvent = Parameters<RequestHandler>[0];
```

This way you actually get a `RequestEvent` definition that accounts for any endpoint-specific types that might be generated by SvelteKit.

Reply:
Thx for the comment. I'll have a look at this type thing.

But other than the type, I get the error above with latest versions of yoga and sveltekit.
(I guess there is something else).

It's working well with yoga 2.0.0. 
Let me know your state plz. 

You might be interested in https://kitql.dev as well 👍